storage/storage_tests/queued_country_tests.cpp

```diff
@@ -97,7 +97,7 @@ UNIT_TEST(QueuedCountry_Bits)
   QueuedCountry country(countryId, MapOptions::MapWithCarRouting);
   TEST_EQUAL(MapOptions::Nothing, country.GetDownloadedFiles(), ());
 
-  TEST(country.SwitchToNextFile(), ());
+  TEST(!country.SwitchToNextFile(), ());
   TEST_EQUAL(MapOptions::Map, country.GetDownloadedFiles(), ());
 
   TEST(!country.SwitchToNextFile(), ());
```

Comment:
Does this test works? How it's possible that SwitchToNextFile() fails 2 times but GetDownloadedFiles() returns different results?